hsm/m0hsm_api.c

```diff
@@ -309,7 +309,7 @@ static int create_obj(struct m0_uint128 id, struct m0_obj *obj,
 	DBG("creating id=%"PRIx64":%"PRIx64"\n", id.u_hi, id.u_lo);
 
 	/* first create the main object with a default layout */
-	m0_obj_init(obj, m0_uber_realm, &id, 9 /* XXX: 1MB */);
+	m0_obj_init(obj, m0_uber_realm, &id, 9 /* XXX: 1MB */, 0);
 			  /* m0_client_layout_id(m0_instance)); */
```

Comment:
```
m0_obj_init(obj, m0_uber_realm, &id, M0_DEFAULT_LAYOUT_ID , 0);
```

Reply:
Shall I change lid passed which was 9 before, to M0_LAYOUT_DEFAULT_ID, which is 1?

Reply:
Sorry, I thought the M0_LAYOUT_DEFAULT_ID was 9.
Let's keep it.

Reply:
Okay.